src/uwu/README.md

```diff
@@ -0,0 +1,15 @@
+---
+home: true
+lang: en-US
+heroImage: /icons/logo-alt.svg
+actionText: Hard installation →
+actionLink: /uwu/getting-started
+features:
+    - title: Paper's back included
+      details: Enter a 100% Paper based new experience in a growing community based on Paper's cult.
+    - title: Native Hentai support
+      details: Discover literature like you never did before.
+    - title: You're an horizontal reader
```

Comment:
"You're a horizontal reader"
`an` doesn't quite fit in a short phrase like this.

```suggestion
    - title: You're a horizontal reader
```